sdk/tx-kernel/src/lib.rs

```diff
@@ -0,0 +1,94 @@
+#![no_std]
+
+#[panic_handler]
```

Comment:
There will be an issue here if these libraries all define a panic handler and global allocator. They are supposed to only be defined in the top-level executable/dylib, and nowhere else in the dependency tree.

So depending on whether the crate is being compiled standalone as a dylib, or as a dependency in rlib form, we need these items, unfortunately there isn't a config flag/feature just for that.

I'm not sure exactly what the best approach is here long term, but I thought in the near term, these crates were going to be pure Rust dependencies, so we don't need to compile them as dylibs on their own, and all of this machinery can be provided by the top-level account/note script crate.

Reply:
Good point! Thanks! Done.